enhancements/ingress/host-network-configurable-ports.md

```diff
@@ -202,11 +206,36 @@ The router gets rid of the loopback hop  by using a Unix domain socket instead o
 
 #### Failure Modes
 
-N/A.
+* A router pod could fail to be scheduled if no node had compatible labels and taints and available ports; in this case, the ingresscontroller's status conditions should report the scheduling failure.
+* A router pod could be scheduled to a node where the configured port were in use by something other than a pod, such as a system daemon (the Kubernetes schedule only checks for conflicts with ports allocated by Kubernetes); in this case, the ingresscontroller's status conditions should reflect the pod's failure to start.
+* A router pod could be scheduled and start listening on a port inside the nodeport range; in this case, a subsequently created nodeport service could by chance be assigned the in-use port and fail to be configured on the host with the router pod.
 
 #### Support Procedures
-n
-N/A.
+
+a. Checking ingresscontroller's status conditions
+
+```sh
+oc get IngressController -n openshift-ingress-operator
+```
+In case of a failure to schedule the router pods, the result should at least show, in the status conditions:
+```yaml
+  - lastTransitionTime: "2022-01-05T12:01:32Z"
+    status: "True"
+    type: PodsScheduled
```

Comment:
In case of failure to schedule pods, the operator would report `PodsScheduled=False` (I hope).